pkgs/ffigen/example/ffinative/headers/example.h

```diff
@@ -16,3 +16,8 @@ float *divide(int a, int b);
 
 /** Divides 2 floats, returns a pointer to double. */
 double *dividePrecision(float a, float b);
+
+const int array[10];
```

Comment:
Default initialization of an object of const type 'const int[10]' clang(default_init_const)

Shouldn't this be initialized if it's a const?

Reply:
I thought it might be fine since it's a header file, but since we're proving a value for the string as well it doesn't hurt to be consistent.